### PR TITLE
graphics: Add two cases about disable fips

### PR DESCRIPTION
**Pull request by cliping**
XXX-103778 - [vnc] Start guest with fips disabled and with vnc password 
XXX-103777 - [vnc] Start guest with fips disabled and without vnc password

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.vnc_only.disable_fips.without_passwd: PASS (13.88 s)
 (2/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.vnc_only.disable_fips.with_passwd: PASS (11.71 s)

**Review by chloerh — APPROVED** [review]

**Review by nanli1 — APPROVED** [review]

File: libvirt/tests/cfg/graphics/graphics_functional.cfg [hunk 1]
```diff
@@ -334,6 +334,16 @@
...
                                     rv_log_auth = "Authentication failed"
+                        - disable_fips:
+                            vnc_autoport = "yes"
+                            check_fips = "yes"
+                            check_qemu_pattern = "-enable-fips"
+                            expect_exist = False
+                            variants:
+                                - without_passwd:
+                                - with_passwd:
+                                    graphic_passwd = 'redhat'
```
Comment (Yingshun) [inline libvirt/tests/cfg/graphics/graphics_functional.cfg:345]: I don't think it's a good idea  to directly show the passwd, i think it's better to update using ci for security issue. what do you think?

**Review by Yingshun — COMMENTED** [review]
Others LGTM

File: libvirt/tests/cfg/graphics/graphics_functional.cfg (hunk 1 above)
Comment (cliping) [reply libvirt/tests/cfg/graphics/graphics_functional.cfg]: Update, thanks.

**Pull request merged**